test/monitor.js

```diff
@@ -402,7 +402,14 @@ describe('good', function () {
             server.register(plugin, function () {
 
                 // .stop emits the "stop" event
-                server.stop();
+                server.stop(
+                    function (err) {
```

Comment:
Please move the function definition up to the same level as `server.stop`. Also, assert the `err` doesn't exist, rather than rethrowing it.